src/main/java/duke/task/Events.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Events extends Task {
+    String time;
+    Date date;
```

Comment:
Modifiers should be private. Use getter methods if it's necessary to access outside class.